src/app/Shared/Services/Api.service.tsx

```diff
@@ -381,10 +405,10 @@ export class ApiService {
     );
   }
 
-  private sendRequest(path: string, config?: RequestInit): Observable<Response> {
+  private sendRequest(path: string, apiVersion: ApiVersion, config?: RequestInit): Observable<Response> {
```

Comment:
Doesn't really matter, but I think having `apiVersion` before `path` might be a bit more readable? The full API paths have the form `/api/v1/foo`, so specifying this as `'/foo', 'v1'` seems maybe backward.

Reply:
> On that note, the `dropzoneProps` property of the `FileUpload` component should probably also be set (see `ArchiveUploadModal` which does this for `.jfr` files) to hint to the user/browser that only certain files should be selectable from the file picker dialog.
I tried getting it to work with multiple extensions but couldn't figure it out, according to the patterfly page and its link to dropzoneProps, the accept could be a string array, but it apparently is only allowed to be a string. Therefore, I didn't want to restrict it to a single extension.